integration-cli/docker_cli_ps_test.go

```diff
@@ -286,6 +286,9 @@ func TestPsListContainersSize(t *testing.T) {
 		t.Fatal(out, err)
 	}
 	lines := strings.Split(strings.Trim(out, "\n "), "\n")
+	if len(lines) != 2 {
```

Comment:
I think it's good this way: we won't hit the error as the test will just stop there.